imap-proto/src/types/mod.rs

```diff
@@ -8,6 +8,9 @@ pub enum AttrMacro {
     Full,
 }
 
+// TODO: why not derive(Copy)?
```

Comment:
No clue. If `derive` works, that would clearly be preferable! This might predate my/the author's understanding of trait derivation.